geniza/footnotes/models.py

```diff
@@ -141,16 +140,49 @@ def __str__(self):
             else:
                 author = author_lastnames[0]
 
-        parts = []
-
         if self.title:
             # if this is an article, wrap title in quotes
             if self.source_type.type == "Article":
                 parts.append('"%s"' % self.title)
             else:
                 parts.append(self.title)
 
-        # TODO: formatted version with italics for book/journal title
+        if self.languages:
+            if not self.languages.all():
+                languages = None
```

Comment:
They way I usually see this logic is to set languages to None outside the conditional, and then set the appropriate value if there is one — it reduces an extra check.

I think I forgot to mention — we only want to include the language in the reference if it is not English.